package.json

```diff
@@ -75,5 +78,6 @@
     "not ie <= 10"
   ],
   "license": "Apache-2.0",
-  "types": "types/index.d.ts"
+  "types": "types/index.d.ts",
+  "packageManager": "yarn@1.22.19"
```

Comment:
Where does this line come fro or is required for?

Reply:
the `packageManager` entry? It's used from nodejs 14 to enforce a package manager version to be used in the project https://nodejs.org/docs/latest-v14.x/api/corepack.html#corepack_configuring_a_package

Since using a newer yarn version to install the dependencies break things we decided with @danimoh to lock it to yarn 1 until we fixed the issues that comes with newer yarn version.

Reply:
It seems it's fixed to a specific version now, not yarn 1 in generell. Is that an issue if I don't have _exactly_ that version?

Reply:
It should not be an issue if you have a lower version, and `1.22.19` is the last yarn 1 version apparently.
Unfortunately I don't think we can provide a more "general" version like `yarn@1.x.x` or `yarn@^1.22.19` since it doesn't seems to be supported yet.

This entry should anyway only be used by node if you have `corepack` enabled, like I do, and installed your package manager using it. And it will tell nodejs to automatically use the correct yarn version in this repo, even if you have yarn 3 globally installed.

Reply:
Ah, thanks! 👍